src/server/acl/validator.cc

```diff
@@ -50,9 +50,8 @@ bool ValidateCommand(const std::vector<uint64_t>& acl_commands, const CommandId&
 
   bool allowed = true;
   if (!pub_sub.all_channels) {
-    for (auto channel : tail_args) {
-      allowed &= iterate_globs(facade::ToSV(channel));
-    }
+    auto channel = tail_args[0];
```

Comment:
Safe, when we reach here we already verified that the command is in a good state (meaning it has exactly two arguments)